ansys/ansys-templates
# Rmorin/udpdate osl solution template with latest capabilities

## What type of PR is this? (check all applicable)


## Description
The purpose of this PR is to update the  to use latest functionality developed to persist data in the UI and use latest changes made to the ui components in the latest solutions-dash-lib and solutions-optislang-frontend-components packages.

## Related Tickets

- Closes https://github.com/Solution-Applications/solution-enablement-product-management/issues/511 and https://github.com/Solution-Applications/solution-enablement-product-management/issues/531

##  Screenshots, Recordings


## QA Instructions,

1. Checkout branch associated to this PR
2. In a .venv run: flit install to gather latest changes made to the osl-solution template
3. Create a new osl-solution using pyowa_oscillator_calibration.owa: ansys-templates new solution -f owa (enter all required inputs)
4. In your terminal, set your current directory to the directory of the new solution.
5.  Setup the python virtual environment by running: python setup_environment.py -d run
6.  Run the solution and create a new project
7. Check that the uploaded input_file is used during analysis
8. Check that the values entered in the problem_setup page are persisted
